docs/sphinx/source/changelog/v3.0.0-beta.0.rst

```diff
@@ -31,6 +31,7 @@ Bug fixes
 * Deploy workflow was replaced with trusted publisher workflow for pypi (:pull:`427`)
 * Fix pandas 2.0.0 deprications and update syntax changes (:pull:`428`)
 * Fix numpy 2.0.0 deprications and update syntax changes (:pull:`428`)
+* Fix `energy_from_power`` returns incorrect index for shifted hourly data (:issue:`370`, :pull:`437`)
```

Comment:
Let's move this to a pending changelog section

Reply:
Done.